controllers/tempest_controller.go

```diff
@@ -188,10 +188,20 @@ func (r *TempestReconciler) reconcileNormal(ctx context.Context, instance *testv
 	instance.Status.Conditions.MarkTrue(condition.ServiceConfigReadyCondition, condition.ServiceConfigReadyMessage)
 	// Generate ConfigMaps - end
 
+	operatorLabels := map[string]string{
+		"operator":  "test-operator",
+		"test-name": instance.Name,
+	}
+
 	serviceLabels := map[string]string{
 		common.AppSelector: tempest.ServiceName,
 	}
 
+	// Combine labels
+	for key, value := range operatorLabels {
+		serviceLabels[key] = value
+	}
+
```

Comment:
Do we need to combine the operatorLabels and serviceLabels like this? Can't we do something like:
```
serviceLabels := map[string]string {
  "operator": "test-operator",
  "test-name": instance.Name,
  common.AppSelector: tobiko.ServiceName
}
```